hs_core/views/__init__.py

```diff
@@ -668,6 +997,37 @@ def get_metadata_terms_page(request, *args, **kwargs):
     return render(request, 'pages/metadata_terms.html')
 
 
+def _send_email_on_group_membership_acceptance(membership_request):
+    """
+    Sends email notification of group membership acceptance
+
+    :param membership_request: an instance of GroupMembershipRequest class
+    :return:
+    """
+
+    if membership_request.invitation_to is not None:
+        # here we are sending email to group owner who invited
+        email_msg = """Dear {}
+        <p>Your invitation to user '{}' to join the group '{}' has been accepted.</p>
+        <p>Thank you</p>
+        <p>The HydroShare Team</p>
+        """.format(membership_request.request_from.first_name,
+                   membership_request.invitation_to.first_name, membership_request.group_to_join.name)
+    else:
+        # here wre are sending email to the user whose request to join got accepted
+        email_msg = """Dear {}
+        <p>Your request to join the group '{}' has been accepted.</p>
+        <p>Thank you</p>
+        <p>The HydroShare Team</p>
+        """.format(membership_request.request_from.first_name, membership_request.group_to_join.name)
+
+    send_mail(subject="HydroShare group membership",
+              message=email_msg,
+              html_message=email_msg,
+              from_email=settings.DEFAULT_FROM_EMAIL,
+              recipient_list=[membership_request.request_from.email], fail_silently=True)
```

Comment:
@hyi Do you see any problem with that? I used similar setting (https://github.com/hydroshare/hydroshare/blob/develop/theme/views.py#L229) as part of another PR review comment previously where I had this attribute set to ```False``` initially.